File: PRAK501-2310817220016-NailaHanifah.py
def MaxBilangan(x, y, z, w):
    max = x
    if y > max:
        max = y
    if z > max:
        max = z
    if w > max:
        max = w
    return max

a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)

File: PRAK501-2310817220016-NailaHanifah.c
#include <stdio.h>

int MaxBilangan(int x, int y, int z, int w){
    int max = x;

    if (y > max){
        max = y;
    }
    if (z > max){
        max = z;
    }
    if (w > max){
        max = w;
    }
    return max;
}

int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    
    return 0;
}